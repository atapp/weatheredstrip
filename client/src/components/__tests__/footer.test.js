import React from 'react';
import { render, cleanup } from 'react-testing-library'
import ReactDOM from 'react-dom';
import Footer from '../footer.react';
import 'jest-dom/extend-expect';
afterEach(cleanup)

it('footer renders properly', () => {
  const { container } = render(<Footer />)
  const footer = container.firstChild
  expect(footer).toMatchSnapshot()
});
