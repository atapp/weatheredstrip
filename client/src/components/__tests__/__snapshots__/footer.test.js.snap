// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders properly 1`] = `
<div
  class="Footer"
>
  <div
    class="Footer-content"
  >
    <div
      class="Footer-item"
    >
      Data taken directly from 
      <a
        href="https://flightplanning.navcanada.ca"
      >
        NAV CANADA
      </a>
      ‚Ñ†
    </div>
    <div
      class="Footer-item"
    >
      <div>
        <a
          href="https://github.com/GregoryHamel/weatheredstrip/issues"
        >
          Report an Issue
        </a>
      </div>
      <div>
        ¬© 
        <a
          href="http://www.greghamel.com"
        >
          Greg Hamel - 
          2019
        </a>
      </div>
    </div>
    <div
      class="Footer-item"
    >
      Made in 
      <span
        aria-label="Canada"
        role="img"
      >
        üçÅ
      </span>
       with 
      <span
        aria-label="love"
        role="img"
      >
        ‚ù§Ô∏è
      </span>
    </div>
  </div>
</div>
`;
